packages/cli/compiler/gz.ts

```diff
@@ -0,0 +1,11 @@
+import { createGzip } from 'node:zlib';
+import { createReadStream, createWriteStream } from 'node:fs';
+import { pipeline } from 'node:stream/promises';
+
+export async function compress(input) {
```

Comment:
im not a fan of utilising the gzipped executables as it adds another fail point to the install script